_schemaV2.graphql

```diff
@@ -9006,6 +9004,9 @@ type Query {
     id: String!
   ): AuctionResult
 
+  # Returns a lot with specific `id`.
+  auctionsLot(id: ID!): AuctionsLotState
```

Comment:
Not going to solve it in this PR but am reminded that we have a sticky situation building up with our naming conventions around 'lots':

- `SaleArtwork` -> gravity's DB/model name for an auction lot
- `Lot` -> Causality's name for an auction lot, lot id is same as gravity SaleArtwork id
- `lot`/`lotState` -> two different names for the same thing in causality's graphql schema. I think we updated the schema to have `lot` be favored and `lotState` is deprecated, reasoning that state was redundant (the type is still `LotState`), but ...
- `Lot` -> Metaphysics' name for the new, authoritative lot type that makes a causality `lot` field (of type `AuctionsLotState`) from causality available on the same object as sale artwork metadata. Thus it is possible to wind up with code that looks like `lot.lot.reserveStatus` which might be confusing.

Only saying this here to observe - if I had it to redo I would have kept the `lotState` version of the name.

Reply:
This is actually `lot`, but what we're doing is prefixing it with the stitched `auctions` prefix

Reply:
Is this automagic? (Like how we append `commerce` to fields from exchange?) If so that's 👍 with me. If we were consciously making a decision here to add `auctions-` then I'd ask we don't add another name 😄.

Reply:
Yup, it indeed is automagic ✨